metaflow/cli.py

```diff
@@ -432,6 +517,9 @@ def step(obj,
                       max_user_code_retries)
 
     echo('Success', fg='green', bold=True, indent=True)
+    # FORMAT: ($1)datastore_root location \t ($2)run_id \t ($3)next_step_to_run \t ($4)task_id(of next step) \t ($5)current step \t ($6)task_id(of current step)
+    if step_name != 'end': # End is the final step
+        print(obj.datastore.datastore_root, run_id, obj.flow._graph[step_name].out_funcs[0], int(task_id)+1, step_name, task_id)
```

Comment:
Is it worth changing this to just be a `logging.debug` command? Looks like it's used for debugging to me.

Reply:
I've added a note to explain why I have it there as Taleb requested in one of his comments, a TODO to remove it when I no longer need it as well. I need this to be there temporarily as I use it to orchestrate via my terminal (i.e., it's not being used for debugging)